docs/contributing/managing-strings.md

```diff
@@ -0,0 +1,181 @@
+3. **Build the project** to ensure there are no references to the removed key remaining.
+
+## 🔀 Merging Weblate PRs
+
+When merging Weblate-generated PRs:
+
+* Check plural forms for **cs, lt, sk** locales. Weblate does not handle these correctly ([issue](https://github.com/WeblateOrg/weblate/issues/7520)).
+* Ensure both `many` and `other` forms are present.
+  * If unsure, reusing values from `many` or `other` is acceptable.
+
+## 🌍 Managing Languages
+
+We use Gradle’s [`androidResources.localeFilters`](https://developer.android.com/reference/tools/gradle-api/8.8/com/android/build/api/dsl/ApplicationAndroidResources#localeFilters%28%29) to control which languages are bundled.
+
+This must stay in sync with the string array `supported_languages` so the in-app picker shows only available locales.
+
+### 🔎 Checking Translation Coverage
+
+Before adding a language, we require that it is at least **70% translated** in Weblate.
+
+We provide a **Translation CLI** script to check translation coverage:
+
+```bash
+./scripts/translation --token <weblate-token>
+
+# Specify the low 60% threshold
+./scripts/translation --token <weblate-token> --threshold 60
+```
+
+- Requires a [Weblate API token](https://hosted.weblate.org/accounts/profile/#api)
+- Default threshold is 70% (can be changed with `--threshold <N>`)
+
+For example code integration, run with --print-all:
+
+```bash
+./scripts/translation --token <weblate-token> --print-all
+```
+
+This output can be used to update:
+
+- `resourceConfigurations` in `app-k9mail/build.gradle.kts` and `app-thunderbird/build.gradle.kts`
+- `supported_languages` in `legacy/core/src/res/values/arrays_general_settings_values.xml`
+
+### ➖ Removing a Language
+
+1. Remove language code from `androidResources.localeFilters` in:
+   * `app-thunderbird/build.gradle.kts`
+   * `app-k9mail/build.gradle.kts`
+2. Remove entry from `supported_languages` in:
+   * `app/core/src/main/res/values/arrays_general_settings_values.xml`
+
+### ➕ Adding a Language
+
+1. Add the code to `androidResources.localeFilters` in both app modules.
+2. Add entry to `supported_languages` in:
+   * `app/core/src/main/res/values/arrays_general_settings_values.xml`
+3. Add corresponding display name in:
+   * `app/ui/legacy/src/main/res/values/arrays_general_settings_strings.xml` (sorted by Unicode default collation order).
+4. Ensure indexes match between `language_entries` and `language_values`.
+
+> [!IMPORTANT]
+> The order of entries in language_entries and language_values must match exactly. Incorrect ordering will cause mismatches in the language picker.
+
+## 🧩 Adding a Component to Weblate
+
+When a new module contains translatable strings, a new Weblate component must be created.
+
+Steps:
+
+1. Go to [Add Component](https://hosted.weblate.org/create/component/?project=3696).
+2. Choose **From existing component**.
+3. Name your component.
+4. For **Component**, select **K-9 Mail/Thunderbird/ui-legacy**.
```

Comment:
Name changed to "Thunderbird for Android / K-9 Mail/legacy-ui-legacy". 

We might need to change this in the future, especially as that component is missing some of the default checks.